### PR TITLE
Create x12-edi-connector-examples.adoc

### PR DESCRIPTION
**Pull request by mcherem**
[Edits] Copy to edit

### CODE REVIEW

File: x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc [hunk 1]
```diff
@@ -0,0 +1,51 @@
+= X12 EDI Connector 2.5 - Examples - Mule 4
+
```
Comment (kbostrom) [inline x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc]: edit instructions:
Type: CE, Content: all, Due date: 2+ days from whenever you start

File: x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+Use this example to view the use case for Business-to-Business (B2B) EDI integration leveraging Anypoint X12 Connector. This example:
+
+* Transforms XML messages into Outbound EDI X12 850 purchase order transactions +
```
Comment (kbostrom) [inline x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc]: Why is "outbound" capitalized?

File: x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+An HTTPS Listener source receives the Enterprise purchase order XML messages from a back-end system. DataWeave and X12 Writes translate the XML message into an X12 v4010 850 Purchase order EDI message.
```
Comment (kbostrom) [inline x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc]: Why are "enterprise," "writes," and "purchase" capitalized?

Comment (kbostrom) [reply x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc]: Per term presentation list, "back-end" > "backend," both as a noun and an adjective.

File: x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+* Transforms Inbound EDI X12 850 purchase orders into JSON +
+
+An HTTP Listener source receives the raw EDI X12 850 purchase order message.
```
Comment (kbostrom) [inline x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc]: Note that "purchase" is not capitalized here, as it is above.

File: x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+X12 Read and DataWeave components pare and translate the EDI data into the JSON format.
+
```
Comment (kbostrom) [inline x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc]: ditto re. "read"; you can init. cap. these if they are understood to be operation names, per style ref.

File: x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+* Generates a Functional Acknowledgment (997) +
+
+== Prerequisites
+
+. Open the Example project in Anypoint Studio from Anypoint Exchange. +
```
Comment (kbostrom) [inline x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc]: Will the audience remember what "the Example project" is...and why capitalized?

File: x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+. Install the latest version of Anypoint X12 connector.
```
Comment (kbostrom) [inline x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc]:
```suggestion
. Install the latest version of Anypoint X12 Connector.
```

File: x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+== Run and Test the Outbound XML to EDI X12 850
+
+. Run the application in Anypoint Studio.
+
+. Test the Outbound flow by using a REST Client to POST a JSON message to the URL:
```
Comment (kbostrom) [inline x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc]: "Outbound" cap?

Comment (kbostrom) [reply x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc]: POST > post: Do not use a name as a verb

File: x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+`http://localhost:8081/x12/send/enterprise-ob-purchaseorder`
+
+Payload location: `src/main/resources/outbound/Enterprise-Outbound-PO.xml`
```
Comment (kbostrom) [inline x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc]: How does this line relate to the narrative above and below it?
Why does it use a colon, and why is it not a complete sentence?

File: x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+
+The Mule app returns the generated EDI X12 850 transaction.
+
+== Run and Test the Inbound EDI X12 850 to JSON
+
+. Test the Inbound flow using a REST Client to POST a raw EDI X12 850 message to the URL:
```
Comment (kbostrom) [inline x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc]: ditto edits line 25

File: x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
++
+`http://localhost:8081/x12/receive/4010-850`
++
+Payload location: `src/main/resources/inbound/EDI-4010-850-NTO-to-Mythical.edi`
```
Comment (kbostrom) [inline x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc]: ditto edit line 29

File: x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
++
+The app returns the translated JSON purchase order message.
++
+A Functional Acknowledgement (997) is generated and sent to the log.
++
+. Send the message back to the trading partner via any standard protocol such as FTP, SFTP, or AS2.
```
Comment (kbostrom) [inline x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc]: comma
```suggestion
. Send the message back to the trading partner via any standard protocol, such as FTP, SFTP, or AS2.
```

File: x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+
+== See Also
+
+* xref:partner-manager::message-flows.adoc[Partner Manager Message Flows].
```
Comment (kbostrom) [inline x12-edi/2.5/modules/ROOT/Edits/x12-edi-connector-examples.adoc]: Remove period
```suggestion
* xref:partner-manager::message-flows.adoc[Partner Manager Message Flows]
```

**Review by kbostrom — DISMISSED** [review]
CE per request

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**